packages/wonder-blocks-button/components/button.md

```diff
@@ -1,78 +1,126 @@
-Primary, secondary, tertiary, and small button examples:
-```js
+There are three `kind`s of buttons: `"primary"` (default), `"secondary"`, and
+`"tertiary"`:
+```jsx
 const {View} = require("@khanacademy/wonder-blocks-core");
 const {StyleSheet} = require("aphrodite");
 
 const styles = StyleSheet.create({
     row: {
         flexDirection: "row",
     },
-    sideMargins: {
+    button: {
         marginRight: 10,
     }
 });
 
 <View style={styles.row}>
     <Button
-        style={styles.sideMargins}
+        style={styles.button}
         onClick={(e) => console.log("Hello, world!")}
-    >Label</Button>
+    >
+        Primary
+    </Button>
     <Button
-        style={styles.sideMargins}
+        style={styles.button}
         onClick={(e) => console.log("Hello, world!")}
         kind="secondary"
-    >Label</Button>
+    >
+        Secondary
+    </Button>
     <Button
-        style={styles.sideMargins}
+        style={styles.button}
         onClick={(e) => console.log("Hello, world!")}
         kind="tertiary"
-    >Label</Button>
-    <Button
-        style={styles.sideMargins}
-        onClick={(e) => console.log("Hello, world!")}
-        size="small"
-    >Label</Button>
+    >
+        Tertiary
+    </Button>
 </View>
 ```
 
-Button example, `href="#button-1"`:
-```js
-<Button
-    onClick={(e) => console.log("Hello, world!")}
-    href="#button-1"
->Label</Button>
+Buttons have a `color` that is either `"default"` (default) or `"destructive"`:
```

Comment:
Might be clearer to say something like `(the default, as shown above) or "destructive" (as can be seen below):`